templates/repo/settings/options.tmpl

```diff
@@ -169,7 +169,11 @@
 						{{range .PushMirrors}}
 						<tr>
 							{{$address := MirrorRemoteAddress $.Context $.Repository .GetRemoteName true}}
-							<td>{{$address.Address}}</td>
+							{{if gt (len  $address.Address) 50}}
+								<td title="{{$address.Address}}">{{slice $address.Address 0 50}}...</td>
```

Comment:
Hmm, wouldn't it be better to always send the whole Address and letting CSS take care of it with overflow?

Reply:
I'm not good as CSS. I know that something like `text-overflow: ellipsis` exists, but I can't get it to look good., so I need a little help with this.

Reply:
Now we have CSS helpers, it just needs `class="gt-word-break"` on the `<td>`

![image](https://user-images.githubusercontent.com/2114189/233968287-177b02d4-bdfb-41ea-add7-88cf25c6a226.png)